MalwareAnalyzer/views/apkid.py

```diff
@@ -1,26 +1,41 @@
 # -*- coding: utf_8 -*-
 import os
+import subprocess
+import json
 import logging
+import shutil
 from django.conf import settings
 from MobSF.utils import filename_from_path
 
 logger = logging.getLogger(__name__)
 
 
-def apkid_analysis(app_dir, apk_file):
+def apkid_analysis(app_dir, apk_file, apk_name):
     """APKiD Analysis of DEX files"""
     apkid_res = {}
     if not settings.APKID_ENABLED:
         return apkid_res
     if not os.path.exists(apk_file):
         logger.error("APKiD - APK not found")
         return {'error': True}
+    dest_dir = os.path.join(app_dir, "apkid/")
```

Comment:
You don't have to use `apkid/`.
Just `apkid` is enough. After all we are using path.join not to worry about the delimiter.

Reply:
done